drivers/tee/optee/call.c

```diff
@@ -433,3 +437,126 @@ void optee_disable_shm_cache(struct optee *optee)
 	}
 	optee_cq_wait_final(&optee->call_queue, &w);
 }
+
+size_t optee_correct_params_cnt(size_t num_params)
```

Comment:
I find this name confusing. What about using `optee_round_up_params_count()` instead?